extensions/PhabBugz/lib/Feed.pm

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::Extension::PhabBugz::Feed;
+
+use 5.10.1;
+use Moo;
+
+use Bugzilla::Extension::PhabBugz::Constants;
+use Bugzilla::Extension::PhabBugz::Util;
+
+has 'is_daemon' => (is => 'rw', default => 0);
+has 'logger'    => (is => 'rw');
+
+sub start {
+    my ($self) = @_;
+    while (1) {
```

Comment:
**optional** 
It might be better to loop in the code that calls this, and also look into https://metacpan.org/pod/Daemon::Generic::While1.

**required**
Meanwhile if this daemon does anything with bugs, it will eventually run out of memory. You at least need to call  Bugzilla->_cleanup() at the end of every run of the loop.